src/baseballgame/Game.java

```diff
@@ -0,0 +1,39 @@
+package baseballgame;
+
+import static camp.nextstep.edu.missionutils.Console.readLine;
+import baseballgame.GameCount;
+
+public class Game extends GameMessage {
+    private static int ChooseNumber(){ // 1, 2 입력
+        String number = readLine();
+        return Integer.parseInt(number);
+    }
+
+    GameCount gameCount = new GameCount();
+
+    public boolean startGame(){
+        System.out.println(RESTART_GAME);
+        int num = ChooseNumber();
+        if(num == 1){
+            return true;
+        }
+        if(num == 2){
+            System.out.println(GAME_EXIT);
+            return false;
+        }
+        throw new IllegalArgumentException(); // 1, 2 말고 다른 숫자 입력했을 때
```

Comment:
다른 수를 입력했을 때의 예외 처리가 좋네요!!

```suggestion
         if(num == START_OPTION){
             return true;
         }
         if(num == EXIT_OPTION){
             System.out.println(GAME_EXIT);
             return false;
         }
         throw new IllegalArgumentException(
                String.format("%d 또는 %d만 입력할 수 있습니다.", START_OPTION, EXIT_OPTION)); 
```

위와 같이 매직넘버를 치환한다면 알아보기 쉬울 것 같아요!
또한 예외 메세지도 같이 넘겨준다면 유저의 접근성이 더 좋지 않을까요!?